Big-Spender/readme.md

```diff
@@ -68,39 +68,39 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+select * from spends where lower (description) like '%fee%';
```

Comment:
nice! Any other way we can do this other than just using lower? 

What if I just want the transactions not every column?